src/main/java/teammates/ui/controller/InstructorFeedbackAbstractAction.java

```diff
@@ -0,0 +1,168 @@
+package teammates.ui.controller;
+
+import java.util.ArrayList;
+import java.util.Arrays;
+import java.util.Collections;
+import java.util.Comparator;
+import java.util.Date;
+import java.util.HashMap;
+import java.util.List;
+import java.util.Map;
+
+import com.google.appengine.api.datastore.Text;
+
+import teammates.common.datatransfer.FeedbackSessionType;
+import teammates.common.datatransfer.attributes.CourseAttributes;
+import teammates.common.datatransfer.attributes.FeedbackSessionAttributes;
+import teammates.common.datatransfer.attributes.InstructorAttributes;
+import teammates.common.util.Const;
+import teammates.common.util.EmailType;
+import teammates.common.util.Logger;
+import teammates.common.util.SanitizationHelper;
+import teammates.common.util.TimeHelper;
+
+public abstract class InstructorFeedbackAbstractAction extends Action {
+
+    private static final Logger log = Logger.getLogger();
+
+    /**
+     * Creates a feedback session attributes object from the request parameters.
+     * @param isCreatingNewSession true if creating a new session; false if editing an existing session.
+     * @return feedback session attributes object.
+     */
+    protected FeedbackSessionAttributes extractFeedbackSessionData(boolean isCreatingNewSession) {
+        // TODO make this method stateless
+
+        // Null checks for parameters not done as null values do not affect data integrity
+
+        FeedbackSessionAttributes newSession = new FeedbackSessionAttributes();
+        newSession.setCourseId(getRequestParamValue(Const.ParamsNames.COURSE_ID));
+
+        // If the class is InstructorFeedbackAddAction then sanitize the newly given title
```

Comment:
The class `InstructorFeedbackAbstractAction` should not know of the existence of `InstructorFeedbackAddAction`. I think the comment can be removed altogether.